ci/properties/rubyonrails.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Ruby on Rails continuous integration",
+    "description": "Build, lint, and test a Rails application",
+    "iconName": "rails",
```

Comment:
This icon needs to be added to https://github.com/actions/starter-workflows/tree/main/icons directory.

Reply:
Done, my apologies. It seems I had a bad gitignore line that prevented me from adding the icon previously.